### PR TITLE
Progress detail

### PR DESCRIPTION
**Pull request by ndeloof**
**- What I did**
Fixed swagger specification according to actual JSON data used by daemon

**- How I did it**
checked pkg/jsonmessage/jsonmessage.go:121 and compared with actual data returned by API

**- How to verify it**
Need to generate API client from swagger spec. Can check https://github.com/dockins/jocker

**- Description for the changelog**
Minor fixes to swagger API

**- A picture of a cute animal (not mandatory but encouraged)**

download

### CODE REVIEW

File: api/swagger.yaml [hunk 1]
```diff
@@ -1496,10 +1496,21 @@ definitions:
         type: "string"
       progressDetail:
         $ref: "#/definitions/ProgressDetail"
+      aux:
+        $ref: "#/definitions/ImageID"
+
+  ImageID:
+     type: "object"
```
Comment (thaJeztah) [inline api/swagger.yaml]:
```
10:38:42 api/swagger.yaml
10:38:42   1503:6    error    wrong indentation: expected 4 but found 5  (indentation)
10:38:44 
```

File: api/swagger.yaml [hunk 2]
```diff
@@ -1496,10 +1496,21 @@ definitions:
...
+     description: "Image ID or Digest"
+     properties:
+       ID:
+         type: "string"
```
Comment (thaJeztah) [inline api/swagger.yaml]: can you add example values to the properties you modified? That way we can (at some point) remove the response/request examples (which can easily get out of sync with the actual definition :smile:)

**Review by thaJeztah — CHANGES_REQUESTED** [review]
looks like there's an issue with indentation; also left a suggestion 👍

File: api/swagger.yaml (hunk 2 above)
Comment (ndeloof) [reply api/swagger.yaml]: ack.
Unfortunately other image related operation use `Id` (lowercase) while buildInfo use uppercase `ID`
https://github.com/moby/moby/blob/master/api/server/backend/build/backend.go#L59

**Review by thaJeztah — COMMENTED** [review]
changes LGTM

can you squash your commits where needed?

**Comment by ndeloof** [comment]
@thaJeztah commits have been squashed.
PR build log on s390 is gone, so can't tell what went wrong there.

**Comment by thaJeztah** [comment]
@ndeloof did you push your change after squashing? I still see three commits 😅 

We can ignore the s390 CI; I saw a message on slack that there's an issue with networking access between Jenkins and IBM's nodes that they're looking in to

**Comment by ndeloof** [comment]
I didn't squashed them all, just grouped them by logical content. Do you prefer a single commit for all of this ?

**Comment by thaJeztah** [comment]
Oh, apologies, I didn't check the actual commits; yes, looks it's fine this way 👍

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Pull request merged**